### PR TITLE
Closes #1858: Fix broken ProboCI 'Check for PHP errors and notices' step.

### LINKED ISSUE #1858: 'Check for PHP errors and notices' ProboCI step is broken
## Problem/Motivation
Our `Check for PHP errors and notices` step in our ProboCI config is not working.  It probably stopped working when we upgraded to Drush 11.

This means PHP errors/warnings/notices generated during the other Probo steps are not being reported by ProboCI.

### Describe the bug
Current output of ProboCI step:
```
$ /src/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:list --type=php

                                                
  The "--type" option does not accept a value.
```
## Proposed resolution
Change ProboCI script to use `drush watchdog:show` instead of `drush watchdog:list`.

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
Our `Check for PHP errors and notices` ProboCI step is not currently working (due to upgrading to Drush 11).

## Related issues
Closes #1858 

## How to test

Look at output of ProboCI `Check for PHP errors and notices` in the ProboCI console.

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [x] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [ ] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [ ] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by trackleft — APPROVED** [review]
Approved https://www.drush.org/latest/commands/watchdog_show/

**Comment by trackleft** [comment]
Do you also want the `--extended` flag on the `drush ws`?

File: .probo.yaml [hunk 1]
```diff
@@ -82,7 +83,7 @@ steps:
       - curl -sS -o /dev/null http://localhost/calendar
       - curl -sS -o /dev/null http://localhost/pages/annual-events
       # Check for PHP watchdog entries
-      - if [ -z "$($SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:list --type=php)" ]; then echo "No PHP notices/errors logged."; else echo "PHP notices/errors found in log." >&2 && exit 1; fi
+      - if [ -z "$($SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:show --count=100 --type=php)" ]; then echo "No PHP notices/errors logged."; else echo "PHP notices/errors found in log." >&2 && exit 1; fi
```
Comment (trackleft) [inline .probo.yaml:86]:
```suggestion
      - if [ -z "$($SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:show --count=100 --type=php --extended)" ]; then echo "No PHP notices/errors logged."; else echo "PHP notices/errors found in log." >&2 && exit 1; fi
```

**Comment by joeparsons** [comment]
> Do you also want the `--extended` flag on the `drush ws`?

We could try it.  The main purpose of the ProboCI check is just to alert us that PHP errors/warnings/notices exist.  They'll likely need to be investigated elsewhere.

**Review by trackleft — DISMISSED** [review]

**Comment by joeparsons** [comment]
This is what the added `drush config:set -y system.logging error_level "verbose"` command does:

Screen Shot 2022-08-12 at 5 15 18 PM

Screen Shot 2022-08-12 at 5 16 20 PM

**Comment by joeparsons** [comment]
> Do you also want the `--extended` flag on the `drush ws`?

*Before:* (31 lines) 
Screen Shot 2022-08-12 at 5 19 22 PM

*After:* (448 lines)
Screen Shot 2022-08-12 at 5 26 09 PM

@trackleft I think I might prefer the non-extended version in this context 🤷🏻‍♂️

File: .probo.yaml [hunk 2]
```diff
@@ -82,7 +83,7 @@ steps:
       - curl -sS -o /dev/null http://localhost/calendar
       - curl -sS -o /dev/null http://localhost/pages/annual-events
       # Check for PHP watchdog entries
-      - if [ -z "$($SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:list --type=php)" ]; then echo "No PHP notices/errors logged."; else echo "PHP notices/errors found in log." >&2 && exit 1; fi
+      - if [ -z "$($SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:show --count=100 --type=php --extended)" ]; then echo "No PHP notices/errors logged."; else echo "PHP notices/errors found in log." >&2 && exit 1; fi
```
Comment (trackleft) [inline .probo.yaml:86]:
```suggestion
      - if [ -z "$($SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -n watchdog:show --count=100 --type=php)" ]; then echo "No PHP notices/errors logged."; else echo "PHP notices/errors found in log." >&2 && exit 1; fi
```
@joeparsons Ok, we can get rid of the `--extended` flag

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**